annotation.go

```diff
@@ -70,7 +70,7 @@ type AnnotationService interface {
 	// ListAnnotations lists all annotations matching the filter.
 	ListAnnotations(ctx context.Context, orgID platform.ID, filter AnnotationListFilter) (ReadAnnotations, error)
 	// GetAnnotation gets an annotation by id.
-	GetAnnotation(ctx context.Context, id platform.ID) (*AnnotationEvent, error)
+	GetAnnotation(ctx context.Context, id platform.ID) (*StoredAnnotation, error)
```

Comment:
For `GetAnnotation` and `GetStream` (below), we need to return the full DB representation of the resource, so that the orgID is available. This is so that the authorization service can verify that the request has read access for that organization.